TypeError: 'where' is an invalid keyword argument for reduce() in CI test

**Gammapy version**
Gammapy 0.15
Numpy 1.16.5
Python 3.7.5

**Bug description**

A number of CI tests fail in `maps/tests/test_wcsnd.py` with "TypeError: 'where' is an invalid keyword argument for reduce()".

**Expected behavior**

The tests succeed

**Actual Behaviour**

When running `python3 -m pytest --pyargs gammapy` on the installed package, I get the following failure:
```
_______________________________ test_to_image _________________________________

geom = WcsGeom

	axes       : ['lon', 'lat', 'energy']
	shape      : (2, 2, 2)
	ndim       : 3
	coordsys   : CEL
	projection : CAR
	center     : 0.0 deg, 0.0 deg
	width      : 1.0 deg x 1.0 deg


    def test_to_image(geom):
        ebounds = np.logspace(-1.0, 1.0, 3)
        axis = MapAxis.from_edges(ebounds, name="energy", unit=u.TeV, interp="log")
        geom = WcsGeom.create(
            skydir=(0, 0), binsz=0.5, width=(1, 1), coordsys="CEL", axes=[axis]
        )
        dataset = MapDataset.create(geom)
    
        # Check map_safe handling
        data = np.array([[[False, True], [True, True]], [[False, False], [True, True]]])
        dataset.mask_safe = WcsNDMap.from_geom(geom=geom, data=data)
    
>       dataset_im = dataset.to_image()

/usr/lib/python3/dist-packages/gammapy/cube/tests/test_fit.py:212: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
/usr/lib/python3/dist-packages/gammapy/cube/fit.py:919: in to_image
    counts = counts.sum_over_axes(keepdims=True)
/usr/lib/python3/dist-packages/gammapy/maps/wcsnd.py:222: in sum_over_axes
    return self.reduce_over_axes(func=np.add, keepdims=keepdims)
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

self = WcsNDMap

	geom  : WcsGeom 
 	axes  : ['lon', 'lat', 'energy']
	shape : (2, 2, 2)
	ndim  : 3
	unit  : 
	dtype : float32

func = <ufunc 'add'>, keepdims = True

    def reduce_over_axes(self, func, keepdims=False):
        """[…]"""
        axis = tuple(range(self.data.ndim - 2))
        geom = self.geom.to_image()
        if keepdims:
            for ax in self.geom.axes:
                geom = geom.to_cube([ax.squash()])
        data = func.reduce(
>           self.data, axis=axis, keepdims=keepdims, where=~np.isnan(self.data)
        )
E       TypeError: 'where' is an invalid keyword argument for reduce()

/usr/lib/python3/dist-packages/gammapy/maps/wcsnd.py:246: TypeError
```

This failure repeats for a number of tests in `test_wcsand.py`. Full test log [here](https://ci.debian.net/data/autopkgtest/testing/amd64/g/gammapy/3623233/log.gz).

Interestingly, I do not see this failure during the [build of the package](https://buildd.debian.org/status/fetch.php?pkg=gammapy&arch=amd64&ver=0.15-1&stamp=1575939456&raw=0). One difference is that the build happens on Debian unstable with numpy 1.17.4, while the test was done on Debian testing, stll with numpy 1.16.5. `setup.cfg` however has 1.16 as minimal version.

**To Reproduce**

On Debian ~~unstable~~ testing

 * install the [gammapy 0.15-1 package](https://packages.debian.org/sid/amd64/python3-gammapy/download)
 * run `python3 -m pytest --pyargs gammapy`

--- a/setup.cfg
+++ b/setup.cfg
@@ -34,7 +34,7 @@ zip_safe = False
 packages = find:
 python_requires = >=3.6
 install_requires =
-    numpy>=1.16
+    numpy>=1.17
     astropy>=3.2
     scipy>=1.2
     regions>=0.4

